### ISSUE TITLE
Mosquitto wont start on Ubuntu 18.04

### ISSUE THREAD

**Issue by AnthonMS** [eclipse/mosquitto#1271]
I know there are so many issues, with this exact topic. But the solutions that works for them, is not working for me. And since they have closed the issues, because they found the solution, I have not had any success.

I have been running Mosquitto on my raspberry pi, with no problems what so ever. But as the Pi is starting to get a bit slow, because it is running so much by now, I though I would upgrade my personel server. So I sat up an older PC, that I was not using anymore, and installed Ubuntu 18.04 on it.

I am now facing the issue A LOT of other people have, and that is, that it is simply not starting up.
The ONLY way I can get this to work, is if I run this command "mosquitto -d" every time I reboot my server. And I do not think that i a viable solution for my problem.

I have tried everything that has been tried on the other closed issues. I have tried giving mosquitto permission to edit log file. I have tried completely removing the log file line from the config. Nothing seems to work.

`Active: failed (Result: exit-code)`
`Failed to start Mosquitto MQTT v3.1/v3.1.1 Broker`

I am hoping someone can help me, and not just close my issue because it looks to much like some of the other issues, that have been solved. Trust me, I have been looking at the other issues for the last 7 hours. And I am loosing my mind here. I sat this up in like 5 mins. on the raspberry pi.....

**Comment by ralight** [comment]
Ok, sorry to hear you're having problems. Let's try and get to the bottom of it. First off, what version of mosquitto are you running, and where was it from? I'm going to work on the assumption that it is the default version provided with ubuntu. That version runs the command `/usr/sbin/mosquitto -c /etc/mosquitto/mosquitto.conf` to start the broker. What happens if you run that command manually? You will probably need to run it with `sudo` to get the same effect as it running from the system.

**Comment by AnthonMS** [comment]
Thank you so much! 
By running that command it showed me this error:
1557428353: Error: Unknown configuration variable "allow".
1557428353: Error found at /etc/mosquitto/mosquitto.conf:13

It let me to the configuration that I tried setting up as I did last time, but for some reason that line was this:
allow anonymous true

I guess that it has been updated, because I am running an older version of Mosquitto on my Raspberry as we speak, and this line does not give me an error on there.

I don't know how I have not stumpled upon that command by googling. But I really appreciate the fast help.
